app/src/main/res/drawable/ic_add_location_black_24dp.xml

```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+        android:width="24dp"
+        android:height="24dp"
+        android:viewportWidth="24.0"
+        android:viewportHeight="24.0">
+    <path
+        android:fillColor="#2F000000"
```

Comment:
These colors should not be set. This is an icon that is imported by android, and colors are set in the component that actually uses the icon. This is simply a default that android generates for you.